.codespell.ignorewords

```diff
@@ -0,0 +1,8 @@
+keypair
+keypairs
+nd
+suh
+od
+rouge
```

Comment:
Nit: Why are we ignoring this one? It's a common misspelling of `rogue`, and I don't think we are referring to the color rouge anywhere...

Reply:
Because of:
```console
$ git grep rouge
.codespell.ignorewords:rouge
site/Gemfile.lock:      rouge (~> 3.0)
site/Gemfile.lock:    rouge (3.12.0)
site/_config.yml:highlighter: rouge
site/_config.yml:  syntax_highlighter: rouge
```

Codespell tries to correct those as following:
```
./site/Gemfile.lock:49: rouge ==> rogue
./site/Gemfile.lock:100: rouge ==> rogue
./site/_config.yml:14: rouge ==> rogue
./site/_config.yml:17: rouge ==> rogue
```

I can ignore entire `site/Gemfile.lock` and `site/_config.yml` files instead if you prefer.

Reply:
> I can ignore entire site/Gemfile.lock and site/_config.yml files instead if you prefer.

I think that's a good move overall to ignore those config type files. 👍

Reply:
Adapted the PR then.